src/main/java/org/broadinstitute/hellbender/engine/FeatureDataSource.java

```diff
@@ -396,7 +397,7 @@ final void printCacheStats() {
             final GenomicsDBExportConfiguration.ExportConfiguration exportConfigurationBuilder =
                     createExportConfiguration(workspace, callsetJson, vidmapJson, vcfHeader, genomicsDBOptions);
             return new GenomicsDBFeatureReader<>(exportConfigurationBuilder,
-                    new GenomicsDBBCFCodec(),
+                    new VCFCodec(),
```

Comment:
As discussed in person, we should benchmark this branch to make sure that the switch to `VCFCodec` did not cause a major performance regression.